pages/communities/communities.md

```diff
@@ -1,43 +1,17 @@
 ---
-title: User Communities
+title: FAIRDOM Community Engagement
 toc: False
 
-redirect_from:
-  - /2016/11/03/digital-salmon-knowledge-base/
 ---
 
-## FAIRDOM supports many communities from different disciplines and domains
+## FAIRDOM Community Engagement
 
-### Systems Biology
+FAIRDOM organizes user meetings twice a year. The purpose of the [FAIRDOM user meetings](https://fair-dom.org/events) is to establish an active FAIRDOM user community to allow networking between various FAIRDOM-SEEK users and to learn from data management experiences in different research projects. 
 
-* [Digital Salmon](https://fairdomhub.org/projects/38)
-* [Plant Systems-biology Modelling](https://fairdomhub.org/projects/129)
+[Biohackathon](https://biohackathon-europe.org) is the hackathon run by ELIXIR, the European ESFRI Infrastructure for Bioinformatics. The website also comprises a list of projects that give an impression. FAIRDOM typically runs or contributes projects in these hackathons. [de.NBI](https://www.denbi.de), the organisation behind the German node of ELIXIR runs a similar set of hackathons (https://www.denbi.de/de-nbi-events).
```

Comment:
I like to include BioHackathon here, but what is meant by "impression" here? Do you mean, the list of projects gives a suggestion of what kinds of projects FAIRDOM is a part of at past BioHackathon events?

Reply:
May be we should rephrase it. FAIRDOM is involved in several projects but I would not list them here separately.

Reply:
[Biohackathon](https://biohackathon-europe.org) is the hackathon run by ELIXIR, the European ESFRI Infrastructure for Bioinformatics. [de.NBI](https://www.denbi.de), the organisation behind the German node of ELIXIR runs a similar set of hackathons (https://www.denbi.de/de-nbi-events). FAIRDOM is typically involved in several projects of these hackathons each year.